cmd/manager/main.go

```diff
@@ -174,3 +185,38 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func initOperatorConfigMapVars(kubeClient client.Client) {
+	builder := &awsclient.Builder{}
+	awsClient, err := builder.GetClient("", kubeClient, awsclient.NewAwsClientInput{
+		SecretName: utils.AwsSecretName,
+		NameSpace:  awsv1alpha1.AccountCrNamespace,
+		AwsRegion:  "us-east-1",
+	})
+
+	// Check if config map exists.
+	cm := &corev1.ConfigMap{}
+	err = kubeClient.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if err != nil {
+		// TODO: Should we just create the configmap here if it doesn't exist?
```

Comment:
My two cents is no.  I would argue the configMap data should be predictable and configuration data should be managed by a human.  If we create one, what is the point of the configMap in general?  We either store the config data in code, or we dont.

Reply:
TODO Removed :)

Thanks @clcollins